netlify.toml

```diff
@@ -21,3 +21,7 @@
 [[redirects]]
   from = "/star"
   to = "https://github.com/trinodb/trino/blob/master/.github/star.png"
+
+[[redirects]]
+  from = "/presto-the-definitive-guide"
```

Comment:
This doesn't work for `presto-the-definitive-guide.html` which is how it's linked today. We should redirect both. You can test with the "deploy-preview" CI check that has a full hosted version of the preview site.

Reply:
Added both .. will test in a sec

Reply:
Both redirects are in place now and work.